tests/test_playbooks/sync_plan.yml

```diff
@@ -46,6 +46,15 @@
       sync_plan_interval: custom cron
       sync_plan_cron_expression: "* * * * */5"
       expected_change: true
+  - include: tasks/product.yml
+    vars:
+      product_sync_plan: Test Sync Plan
+      product_state: present
+      expected_change: true
+  - include: tasks/sync_plan.yml
+    vars:
+      sync_plan_state: present_with_defaults
```

Comment:
shouldn't this test pass some parameter that won't be updated? like the product list or something?

Reply:
Yes, you are right, and it permit me to find a bug. Changed.